### ISSUE TITLE
Conceptual problems in implementation

### ISSUE THREAD

**Issue by ChaitanyaGoyal** [ChaitanyaGoyal/Linear_Elasticity_Cantilever_Beam#1]
1. The beam mesh is being imported from gmsh to Fenics and hence has be converted and tagged accordingly for facets and subdomains. The way I import the converted files and use them in the main equation along with the Boundary conditions is what's confusing me.

2. How to physically understand the main variational equation a=L (in this case the linear elasticity equation) is a problem. In the Fenics documentation shown below: http://fenicsproject.org/pub/documents/fenics/presentations/logg-geilo-2006-03/logg-geilo-2006.pdf
Here, on Pg. 69, the equations are defined as 

a = dot(grad(v), sigma(U))*dx
L = dot(v, f)*dx

Then, I came across this example of solving PDEs with subdomains and to apply force selectively where you want.

http://fenicsproject.org/documentation/dolfin/dev/python/demo/pde/subdomains-poisson/python/documentation.html

According to what they did above with Poisson eqn., for elasticity I would need to define boundaries as 'ds' and tag them to apply force or displacement selectively on a boundary. So we would represent this in variational form as:

a = dot(grad(v), sigma(U))*dx
L = dot(v, f)*ds(tagging number for the facet)

I understand that DOLFIN predefines the “measures” dx, ds and dS representing integration over cells, exterior facets (that is, facets on the boundary) and interior facets, respectively. So what does this elasticity equation do exactly with dx and ds?

I used this approach successfully for a beam mesh defined in Fenics itself (code posted in repository). However, things are not clear when importing the mesh from gmsh.

3. To apply a point load on free end, I defined one of the corner vertices on the free end of beam as a 'physical point' in gmsh. I also defined the edge on free end as a 'physical line' to try loading an edge. Now, according to what I understand, this should have worked, but it doesn't.

ds = Measure("ds")[boundaries]
a = inner(grad(v),sigma)*dx 
L= dot(v,f)*ds(point tag)        <-- i.e. the number for point tag or edge tag

To sum up the story, I want to understand, how the equation behaves with dx and ds and how to successfully imply load on a point, load on an edge and load on the top face of the cantilever beam. All codes including geo file are posted.
